src/test/java/uk/gov/hmcts/reform/civil/sampledata/CaseDataBuilder.java

```diff
@@ -192,9 +195,14 @@
     private LocalDate interestFromSpecificDate;
     private BigDecimal breakDownInterestTotal;
     protected LocalDateTime respondent1LitigationFriendDate;
+    protected LocalDateTime respondent2LitigationFriendDate;
+    protected LocalDateTime genericLitigationFriendDate;
```

Comment:
This field should never be required - as we only set Dates for Respondent 1 and 2